### PR TITLE
chorus : bump versions for 2.1.13 release

### PR DESCRIPTION
**Pull request by aureamunoz**
Could you guys, please, take a look? Thx!

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -179,18 +179,19 @@
     <narayana-starter.version>2.1.1</narayana-starter.version>
     <cxf-spring-boot-starter-jaxrs.version>3.3.4</cxf-spring-boot-starter-jaxrs.version>
     <vertx-spring-boot.version>0.0.8</vertx-spring-boot.version>
-    <dekorate.version>0.10.10</dekorate.version>
+    <dekorate.version>0.11.2</dekorate.version>
+    <netty.version>0.8.16.RELEASE</netty.version>
 
     <!-- Spring Ecosystem -->
-    <spring-boot.version>2.1.12.RELEASE</spring-boot.version>
+    <spring-boot.version>2.1.13.RELEASE</spring-boot.version>
```
Comment (gytis) [inline pom.xml]: I thought we're staying on 2.1.12?

Comment (gytis) [reply pom.xml]: Otherwise, there shouldn't be a need for Netty dependency.

Comment (aureamunoz) [reply pom.xml]: No, we should release the `2.1.13`

File: pom.xml [hunk 2]
```diff
@@ -534,6 +535,12 @@
         <version>${undertow.version}</version>
       </dependency>
 
+      <dependency>
```
Comment (gytis) [inline pom.xml]: We don't need this if we upgrade to Spring Boot 2.1.13. It has the CVE fix already.

**Comment by geoand** [comment]
@iocanel should we be using the latest dekorate? IIRC you fixed some bugs

**Comment by cmoulliard** [comment]
@aureamunoz Please update the BOM versions of the components as defined within the G Sheet as some versions are not aligned at all with the G Sheet - 2.1.13. 

Example : 
- Infinispan - bom version : 9.4.16.Final BUT G sheet: 9.4.15.Final
- Tomcat - bom version : 9.0.21 BUT G sheet : 9.0.31
....

File: pom.xml (hunk 2 above)
Comment (aureamunoz) [reply pom.xml]: ah!!! ok ok

Comment (gytis) [reply pom.xml]: I thought we're staying on 2.1.12, then we would have had to override the reactor-netty.

**Comment by aureamunoz** [comment]
> @iocanel should we be using the latest dekorate? IIRC you fixed some bugs
I took `0.11.2` because I saw that was the latest [here](https://github.com/dekorateio/dekorate/releases)

**Comment by geoand** [comment]
@iocanel we will need the snowdrop tag released as well (if it's not already done)

**Comment by cmoulliard** [comment]
LGTM, versions verified. Can be merged @metacosm

**Pull request merged**